sites/platform/src/learn/overview/yaml/what-is-yaml.md

```diff
@@ -75,10 +115,29 @@ web:
             allow: false
 ```
 
+<--->
+<!-- Upsun -->
+
+```yaml {configFile="app"}
+applications:
+    myapp:
+        type: "golang:1.18"
+
+        web:
+            commands:
+                start: ./bin/app
+            locations:
+                '/':
+                    passthru: true
+                    allow: false
+```
+{{% /version/specific %}}
+
 This creates a `web` dictionary that has two dictionaries within it: `commands` and `locations`,
-each with their own mappings.
+each with their own mappings:
 
-![web mappings](/images/yaml/mapping.svg)
+- `web` → `commands` → `start: ./bin/app`
```

Comment:
@chadwcarlson I know this may not be ideal, but I didn't want to waste time on creating visuals when we have so little time left for the launch. I think it's ok for the time being, especially as I don't think this page is of paramount importance, and we can improve it later on. WDYT?